tracer/src/Datadog.Trace/Ci/ImpactedTestsModule.cs

```diff
@@ -45,10 +44,10 @@ private ImpactedTestsModule(string baseCommitSha, string currentCommitSha, FileC
     /// <summary>
     /// Creates an instance of the <see cref="ImpactedTestsModule"/> class.
     /// </summary>
-    /// <param name="impactedTestsDetectionResponse">Impacted tests detection backend response</param>
     /// <param name="environmentValues">CI environment values</param>
+    /// <param name="defaultBranch">Repository default branch</param>
     /// <returns>ImpactedTestModule instance</returns>
-    public static ImpactedTestsModule CreateInstance(TestOptimizationClient.ImpactedTestsDetectionResponse impactedTestsDetectionResponse, CIEnvironmentValues environmentValues)
+    public static ImpactedTestsModule CreateInstance(CIEnvironmentValues environmentValues, string? defaultBranch)
```

Comment:
Not a super big fan of passing the `string? defaultBranch` here, do we have a better option here? cc @tonyredondo